app/src/main/java/com/example/footballmanager/ui/screens/AccountScreen.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.footballmanager.ui.screens
+
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+
+@Composable
```

Comment:
It's overkill to wrap Text in the other function

Reply:
@PawelZ4krzewski It's just a screen mockup

Reply:
@PawelZ4krzewski It's just a screen mockup